src/OpenIddict.Core/Infrastructure/OpenIddictProvider.Serialization.cs

```diff
@@ -17,14 +17,41 @@
 namespace OpenIddict.Infrastructure {
     public partial class OpenIddictProvider<TUser, TApplication, TAuthorization, TScope, TToken> : OpenIdConnectServerProvider
         where TUser : class where TApplication : class where TAuthorization : class where TScope : class where TToken : class {
-        public override async Task SerializeRefreshToken([NotNull] SerializeRefreshTokenContext context) {
+        public override async Task SerializeAuthorizationCode([NotNull] SerializeAuthorizationCodeContext context) {
```

Comment:
ASOS <beta6 used to support automatic authorization code revocation by storing codes in a distributed cache and returning a corresponding cache key. This is no longer true with beta6 (due to the completely stateless approach) so OpenIddict is now responsible of implementing authorization code revocation to prevent code reuse.

Since we already have the infrastructure we need for that (`OpenIddictTokenManager`), we're just leveraging it here.